### ISSUE TITLE
A large donation stuck in limbo (donated via mycrypto)

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapp#491]
https://etherscan.io/tx/0xab4751921b47b52127d2792e124adec88c8ca079851d7058a12fc72dcc291cf3

successful donation :-D

But the UI is freaking out...

It isnt showing up in https://beta.giveth.io/delegations where the rest of the decentral donations are showing up so I can't delegate it yet.. 

but it shows up in the campaign ( https://beta.giveth.io/campaigns/5b3d94ba329bc64ae74d1421 ) as donated twice once failed once completed:
<img width="906" alt="screen shot 2018-10-10 at 12 07 21 am" src="https://user-images.githubusercontent.com/14945613/46701753-169a5400-cc21-11e8-86f3-a859f34a9234.png">

Also shows up under https://beta.giveth.io/donations when logged into the DECENTRAL Multisig account...
<img width="984" alt="screen shot 2018-10-10 at 12 08 20 am" src="https://user-images.githubusercontent.com/14945613/46701964-d25b8380-cc21-11e8-9a46-7ae1fe4f5949.png">

Its worth mentioning that this is the second 100+ ETH donation frozen in limbo in the last few days, I assume that the size of the donation doesn't matter for the dapp... but maybe? 

Also I used MyCrypto for this donation... that shouldnt matter either... but maybe?

:-D

**Comment by vojtechsimetka** [comment]
I think it is just unlucky. But using MyCrypto instead of metamask might be relevant as we actually store some donation data before it hits the chain when donating through metamask (and we clearly can't do that with mycrypto/mew).

**Comment by perissology** [comment]
Are you sure this was donated via mycrypto?

It looks like the donation was originally made via metamask & the ui & then possibly canceled (via re-using a nonce) and donated again.

The reason I'm saying this is the "Failed" donations has only a `homeTx` (mainnet) hash and no rinkeby txHash. This should only be the case when donating via the ui. The 'homeTx` hash doesn't exist anymore, making me think the tx was replaced

**Comment by GriffGreen** [comment]
@perissology yes exactly, I tried to make the donation in Metamask... but then there was a nonce issue so i replaced the tx with MyCrypto

background:

 I still have the nonce issue in fact (maybe i should file a bug for MM ;-) ) but its ok cause its a throw away account used only for moving the DECENTRAL multisig funds to the dapp

The bug in MM happened because i was donating 3 and 4 at a time in other people's names (so i could utilize the tracking features of our awesome app! I collected money before it was easy to let people donate directly to the dapp so i was manually putting in that data) and i think i was going to fast for metamask and it let me send a transaction for more ETH then i ever had... and that was clogging up my nonce! (see 32 in this pic)

![screen shot 2018-10-10 at 8 33 05 pm](https://user-images.githubusercontent.com/14945613/46758079-2fac0f00-cccc-11e8-87df-28a8bd695014.png)


So since that account was blocked from being used by MM (every time i tried it was giving me a nonce that was too big as you can see in the pic) I used mycrypto to manually choose the nonce and push it thru... replacing nonce 32

**Comment by vojtechsimetka** [comment]
To fix the nonce just change the network to localhost and back. That should refresh your nonces :).

**Comment by perissology** [comment]
so the current issue (besides the "Failed" donation being shown & a bit confusing) is:

The donation is visible via the 'my-donations' page for the 'decentral multisig' user but the donation does not appear in the 'my-delegations' page for the "griff green" user (are you sure you are using the correct account for the delegation?)

**Comment by GriffGreen** [comment]
Yes @perissology I am sure, as it was donated to the DECENTRAL campaign... and i can delegate the other donations i made to that campaign

These pics are from today:

Here you can see that the donation happened... tho there is some failed thing still going on...
<img width="980" alt="screen shot 2018-10-13 at 2 04 24 am" src="https://user-images.githubusercontent.com/14945613/46905286-a413c780-cef1-11e8-9488-2c4212b2dc9f.png">

So it's donated and I'm pretty sure you already ran thru the events to fix this when i first mentioned it above, cause the DAC money is able to be played with...

Here you can see all of my delegations (and no 133 donation)... maybe i have too many donations and the list is being cut short? 

<img width="1213" alt="screen shot 2018-10-13 at 2 05 00 am" src="https://user-images.githubusercontent.com/14945613/46905285-a413c780-cef1-11e8-9ce8-aa9542f417e7.png">


<img width="1227" alt="screen shot 2018-10-13 at 2 05 08 am" src="https://user-images.githubusercontent.com/14945613/46905284-a413c780-cef1-11e8-9971-ba3547d5bb87.png">

<img width="1215" alt="screen shot 2018-10-13 at 2 05 17 am" src="https://user-images.githubusercontent.com/14945613/46905283-a37b3100-cef1-11e8-91f5-366256bf05e7.png">

**Comment by GriffGreen** [comment]
I'm playing these games to test the dapp, mostly as a user story, and slightly to bring transparency to the burn camp.... thats why i have all these 3 and 4 ETH donations...

**Comment by vojtechsimetka** [comment]
Nice!
